tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -191,6 +188,16 @@ func (s *stresser) Stress() error {
 	}
 	s.stressTable = createStressTable(stressEntries)
 
+	return nil
+}
+
+func (s *stresser) Stress() error {
+	if err := s.Setup(); err != nil {
```

Comment:
I removed setup() from interface but kept setup() for stresser to break down the stress() function a bit, making it more decoupled.